mcs/class/System/Mono.Net.Security/MonoTlsProviderFactory.cs

```diff
@@ -192,6 +193,25 @@ static MSI.MonoTlsProvider LookupProvider (string name, bool throwOnError)
 			}
 		}
 
+#if MARTIN_TLS_DEBUG
+		static bool enableDebug;
+
+		static void InitializeDebug ()
```

Comment:
you could just add the `[Conditional ("MARTIN_TLS_DEBUG")]` here as well, then you don't need the `#if MARTIN_TLS_DEBUG`